schemas/remote_configuration_v2_template.schema.json

```
@@ -0,0 +1,41 @@
+{
+  "$id": "remote_configuration_v2_template",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "Config template files stored as object which contain list of conditional gathering rules and list of container logs",
+  "example": {
+    "conditional_gathering_rules": [
+      "conditional_gathering_rules/*.json",
+      "conditional_gathering_rules/rule1.json"
+    ],
+    "container_logs": [
+      "container_log_requests/ccx-rules-ocp/*.json",
+      "container_log_requests/manual/kube_controller_manager_logs_gatherer.json"
+    ]
+  },
+  "properties": {
+    "conditional_gathering_rules": {
+      "description": "Array of glob patterns matching conditional gathering rules to be included in the configuration file. The glob patterns are relative to the source data (repository) root.",
+      "items": {
+        "description": "Path to conditional gathering rule",
```

Should this be updated as well?

```suggestion
        "description": "A glob pattern selecting conditional gathering rules",
```